### PR TITLE
Add sound when pv enters alarm state

### PR DESCRIPTION
**Pull request by nstelter-slac**
use print ('\a') to produce cross-platform sound when any alarm enters active-alarm state

also add option for short-hand cmd line args (-a and --annunciate as options)

### CODE REVIEW

**Comment by nstelter-slac** [comment]
using debug print statement to verify when it should make noise atm, need to test that the actual sound effect is outputted too

**Comment by nstelter-slac** [comment]
need to finish new test

also bug occurs only when running on cryo: setting annunciate on a pv with active alarm removes the alarm ???

**Comment by nstelter-slac** [comment]
to try feature:
on dev-rhel7 machine:
-slam --topics LCLS --bootstrap-servers 172.24.5.224:9094 -a
-in left-hand tree menu, click on "LI24->SBI->SBI:LI24:1:SBI_ACC_NS" and check "Annunciate" and then "Ok"
-since this alarm continually toggles in/out of active-alarm state, you should hear a "beep" noise play as the alarm toggles 

(note: beep will not play if you have "Terminal Bell" disabled in the terminal that launched slam (or possibly if other "system sounds" settings are disabled))

File: slam/alarm_table_view.py [hunk 1]
```diff
@@ -61,6 +61,7 @@ def __init__(
         topic: str,
         table_type: AlarmTableType,
         plot_slot: Callable,
+        annunciate: bool = False,
```
Comment (jbellister-slac) [inline slam/alarm_table_view.py:64]: not currently used for tables but maybe could be useful in the future

File: slam_launcher/main.py [hunk 2]
```diff
@@ -8,14 +8,18 @@
...
         default="localhost:9092",
         help="Comma separated list of urls for one or more kafka boostrap servers",
     )
-    parser.add_argument("--user-permissions", default="admin", help="One of read-only, operator, admin")
-    parser.add_argument("--log", default="warning", help="Logging level. debug, info, warning, error, critical")
+    parser.add_argument("-u", "--user-permissions", default="admin", help="One of read-only, operator, admin")
+    parser.add_argument("-l", "--log", default="warning", help="Logging level. debug, info, warning, error, critical")
+    parser.add_argument(
+        "-a", "--annunciate", action="store_true", help="Enable beep from alarms that have annunciate setting enabled"
+    )  # default=False
```
Comment (jbellister-slac) [inline slam_launcher/main.py:22]: thanks for the easier command line options

**Review by jbellister-slac — APPROVED** [review]
Looks good, and like discussed can follow up with a choice of different sounds later if needed, thanks!

**Pull request merged**